internal/helper/loadbalancermachine_test.go

```diff
@@ -310,3 +311,35 @@ makestep 1 3
 `))
 	})
 })
+
+var _ = DescribeTable("GetHashForLoadBalancerMachineSet",
+	func(lb *yawolv1beta1.LoadBalancer, expectedHash string) {
+		hash, err := GetHashForLoadBalancerMachineSet(lb)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(hash).To(Equal(expectedHash))
+	},
+	Entry("empty config", &yawolv1beta1.LoadBalancer{}, "ogwzb3hasmo2o2tj"),
+	Entry("basic config", &yawolv1beta1.LoadBalancer{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "test",
+			Namespace: "testspace",
+		},
+		Spec: yawolv1beta1.LoadBalancerSpec{
+			DebugSettings: yawolv1beta1.LoadBalancerDebugSettings{
+				Enabled:    true,
+				SshkeyName: "key",
+			},
+			Options: yawolv1beta1.LoadBalancerOptions{
+				LogForward: yawolv1beta1.LoadBalancerLogForward{
+					Enabled: true,
+					LokiURL: "url",
+				},
+				ServerGroupPolicy: "affinity",
+			},
+		},
+		Status: yawolv1beta1.LoadBalancerStatus{
```

Comment:
Can we add another use case that should produce the same hash as "basic config" but has different fields, for example in the status?

Reply:
I've added a test case that sets a few extra fields.